qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ProviderSteps.java

```diff
@@ -60,7 +60,7 @@ public void launchSystemAdministrationPage() {
     
     @Then ("system loads administration page")
     public void systemLoadsAdministrationPage() {
-    	assertPage(administrationPage.waitForPage());
+    	assertTrue(textExists("Advanced Administration"));
```

Comment:
which text appears when the system loads `System Administration` page?

Reply:
i get the same error as the one on the build failures when i use `assertTrue(textExists("System Administration"));`

Reply:
> when i use `assertTrue(textExists("System Administration"));`

could you use the suggested text to see the build result on github actions?

Reply:
sorry saw this late after pushing that commit

Reply:
but `assertTrue(textExists("Administration"));` passes locally